Support for polar scatterplots

Carifio24 opened the pull request:
This PR adds support for exporting polar scatterplots. This is accomplished by using the `plotly.graph_objs.layout.Polar` layout.

The tick marks for the radial axis are grabbed directly from the scatter viewer and italicized via HTML tags (as best as I can see, there's no option to directly italicize the font). For the theta axis, the formatting used to create the pi symbol in the ticks in matplotlib doesn't work in plotly, so rather than try some sort of text replacement hack, this PR just lets plotly use its own tick marks and adds a suffix for the theta=0 marker.

Some examples of the output are [here](https://drive.google.com/drive/folders/1exXR0wGmgSSfHv4u-Hnyt_1Fb3XZagMO?usp=sharing). Most of these are using the W5 dataset, but the output of the polar plot used in the airplane video is in there as well (that file takes a few seconds to load).

astrofrog reviewed (changes requested):
This looks good generally, but could you add a test to at least make sure the code runs (even if we can't really check the output) and also add a changelog entry?

Carifio24 commented:
I've added a changelog entry, as well as tests using both radians and degrees that follow the same pattern as the rectilinear test. I refactored the export of the figure into its own function since it would've been essentially identical in all three tests.

I should note that the polar plots in the test output don't have radial axis labels. This is because the exporter directly uses the axis labels for the radial axis, which seem to all be empty when running glue via a call to`GlueApplication()` like we do in the test.
